<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1270</width>
    <height>688</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QLabel" name="label_8">
     <property name="font">
      <font>
       <family>Agency FB</family>
       <pointsize>26</pointsize>
      </font>
     </property>
     <property name="text">
      <string>机械臂控制系统</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0" colspan="2">
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>前</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>舵机1</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3" colspan="2">
      <widget class="QPushButton" name="pushButton_3">
       <property name="text">
        <string>上</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="2">
      <widget class="QLineEdit" name="lineEdit_3"/>
     </item>
     <item row="2" column="4" colspan="2">
      <widget class="QLineEdit" name="lineEdit_4"/>
     </item>
     <item row="1" column="5">
      <widget class="QPushButton" name="pushButton_4">
       <property name="text">
        <string>下</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QPushButton" name="pushButton_5">
       <property name="text">
        <string>弱</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="pushButton_6">
       <property name="text">
        <string>强</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3" colspan="2">
      <widget class="QPushButton" name="pushButton_7">
       <property name="text">
        <string>右</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>舵机4</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>后</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>舵机2</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>舵机3</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4" colspan="2">
      <widget class="QLineEdit" name="lineEdit_2"/>
     </item>
     <item row="3" column="5">
      <widget class="QPushButton" name="pushButton_8">
       <property name="text">
        <string>左</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>434</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>444</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="pushButton_13">
     <property name="text">
      <string>连接服务器</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>478</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
